package proto_ff;

import "yd_fieldoptions.proto";


message E_TeamTeam
{
	optional int32 m_TeamObjectID = 1[(yd_fieldoptions.field_cname) = "组队目标ID"];
	optional int32 m_FObjectNameID = 2[(yd_fieldoptions.field_cname) = "父目标名称ID"];
	optional int32 m_SObjectNameID = 3[(yd_fieldoptions.field_cname) = "子目标名称ID"];
	optional int64 m_duplicateId = 4[(yd_fieldoptions.field_cname) = "副本ID"];
	optional int64 m_activityId = 5[(yd_fieldoptions.field_cname) = "活动ID"];
	optional int32 m_LowLV = 6[(yd_fieldoptions.field_cname) = "下限等级"];
	optional int32 m_UpLV = 7[(yd_fieldoptions.field_cname) = "上限等级"];
	optional int32 m_isAutoMatch = 8[(yd_fieldoptions.field_cname) = "是否自动匹配"];
	optional int32 m_speakChannel = 9[(yd_fieldoptions.field_cname) = "喊话频道"];
	optional int32 m_teamnumber = 10[(yd_fieldoptions.field_cname) = "序列"];
}


message Sheet_TeamTeam
{
	repeated E_TeamTeam E_TeamTeam_List = 1[(yd_fieldoptions.field_arysize)=64];
}
